install/install.php

```diff
@@ -502,7 +506,7 @@ function step8() {
    $url_base = str_replace("/install/install.php", "", $_SERVER['HTTP_REFERER']);
    $DB->update(
       'glpi_configs',
-      ['value' => $DB->escape($url_base)], [
+      ['value' => $DB->quote($url_base)], [
```

Comment:
why the `quote` needed here ?

Reply:
This is a mistake, queries parameters must not be quoted